charts/karpenter/templates/clusterrole-core.yaml

```diff
@@ -41,9 +41,11 @@ rules:
   - apiGroups: ["apps"]
     resources: ["daemonsets", "deployments", "replicasets", "statefulsets"]
     verbs: ["list", "watch"]
-  - apiGroups: ["apiextensions.k8s.io"]
-    resources: ["customresourcedefinitions"]
-    verbs: ["watch", "list"]
+{{- if .Values.webhook.enabled }}
+  - apiGroups: ["admissionregistration.k8s.io"]
+    resources: ["validatingwebhookconfigurations", "mutatingwebhookconfigurations"]
```

Comment:
shouldn't need to add these back in. Should keep the old rbac permissions, but just gate them with the flag.